docs/basics/101-130-yodaproject.rst

```diff
@@ -654,48 +655,72 @@ configure this repository as a sibling of the dataset:
-   publish(ok): . (dataset) [pushed to github: ['[new branch]', '[new branch]']]
-   action summary:
-     publish (ok: 1)
+.. note::
 
-.. gitusernote::
+    Publishing is one of the remaining big concepts that this handbook tries to
+    convey. However, publishing is a complex concept that encompasses a large
+    proportion of the previous handbook content as a prerequisite. In order to be
+    not too overwhelmingly detailed, the upcoming sections will approach
+    :command:`push` from a "learning-by-doing" perspective:
+    You will see a first :command:`push` to GitHub below, and the findoutmore at
+    the end of this section will already give a practical glimpse into the
+    difference between annexed contents and contents stored in Git when pushed
+    to GitHub. The chapter :ref:`chapter_thirdparty` will extend on this,
+    but the section
+
+    .. todo::
+
+       write and link
+
+    will finally combine and link all the previous contents to give a comprehensive
+    and detailed wrap up of the concept of publishing datasets. In this section,
+    you will also find a detailed overview on how :command:`push` works and which
+    options are available. If you are impatient or need an overview on publishing,
+    feel free to skip ahead. If you have time to follow along, reading the next
+    sections will get you towards a complete picture of publishing a bit more
+    small-stepped and gently.
+    For now, we will start with learning by doing, and
+    the fundamental basics of :command:`datalad push`: The command
+    will make the last saved state of your dataset available (i.e., publish it)
+    to the :term:`sibling` you provide with the ``--to`` option.
+
+.. runrecord:: _examples/DL-101-130-118
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
 
-   The :command:`datalad publish` uses ``git push``, and ``git annex copy`` under
-   the hood. Publication targets need to either be configured remote Git repositories,
-   or git-annex special remotes (if they support data upload).
+   $ datalad push --to github
 
-Here is one important detail, though: By default, your tags will not be published.
+Thus, you have now published your dataset's history to a public place for others
+to see and clone. Below we will explore how this may look and feel for others.
+There is one important detail first, though: By default, your tags will not be published.
+Thus, the tag ``ready4analysis`` is not pushed to GitHub, and currently this
+version identifier is unavailable to anyone else but you.
 The reason for this is that tags are viral -- they can be removed locally, and old
 published tags can cause confusion or unwanted changes. In order to publish a tag,
-an additional :command:`git push` with the ``--tags`` option to the
-sibling would be required:
+an additional :command:`git push` (yes, we need to push with Git, not DataLad here)
+with the ``--tags`` option is required:
```

Comment:
Maybe mention that the idea for datalad would be to configure a push configuration with git config settings, such as a definition of what kind of tags are to be pushed.

Reply:
done